From ad2bf0e5856f5566085c612ca2134e195b48f8ef Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 04 Jun 2019 18:43:46 +0300
Subject: [PATCH] [SW-0] gaudi: pll: (re)use common clk sel offset

The commit:
a8a4753 ("[SW-4185] gaudi: pll: add hbw clk sw glitch w/a")
mistakenly added the HL_PLL_DIVSEL_OFFSET macro while in fact it already
exists in the pll.h header.
Remove the private macro and (re)use the one from common pll.h header.

Change-Id: I1b21ff685d95bde441a70bee431f3aa05b8d88a0
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/asic.c b/gaudi/asic.c
index acd3c6c..f6bc30c 100644
--- a/gaudi/asic.c
+++ b/gaudi/asic.c
@@ -342,13 +342,12 @@
 }
 
 #define HL_PLL_GATER_OFFSET	0x300
-#define HL_PLL_DIVSEL_OFFSET	0x280
 
 static void hl_pll_hbw_gated_clk_sw(struct hl_pll_cfg *cfg)
 {
 	regmap_writel(0xE, cfg->base + HL_PLL_GATER_OFFSET);
 	hl_pll_udelay(1);
-	regmap_writel(HL_PLL_PLL_CLK, cfg->base + HL_PLL_DIVSEL_OFFSET);
+	regmap_writel(HL_PLL_PLL_CLK, cfg->base + HL_PLL_DIV_SEL);
 	hl_pll_udelay(10);
 	regmap_writel(0xF, cfg->base + HL_PLL_GATER_OFFSET);
 	hl_pll_udelay(10);
